build-js.sh

```diff
@@ -44,7 +44,7 @@ EMCC_ARGS="$EMCC_ARGS -s NO_FILESYSTEM=0"
 EMCC_ARGS="$EMCC_ARGS -s WASM=0"
 EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
 EMCC_ARGS="$EMCC_ARGS -s BINARYEN_ASYNC_COMPILATION=0"
-# TODO: enable this (need nearbyint in emscripten tag) EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
+EMCC_ARGS="$EMCC_ARGS -s ERROR_ON_UNDEFINED_SYMBOLS=1"
```

Comment:
This is (a) already two lines above and (b) the default, so these lines should both be removed.

Reply:
Nice catch, totally missed that. Did the default of `ERROR_ON_UNDEFINED_SYMBOLS` change recently? Iirc it didn't use to be enabled by default.

Reply:
It changed a while ago.  Looks like v1.38.13